backend/.env

```
@@ -1,6 +1,11 @@
 PORT=5000
-MONGODB_URI=mongodb://localhost:27017/blockchain-document-verification
+MONGO_URI=mongodb://localhost:27017/docverify
 JWT_SECRET=your_super_secret_jwt_key_here_make_it_long_and_secure
+SESSION_SECRET=your_super_secret_session_key_make_it_different_from_jwt
 JWT_EXPIRES_IN=7d
 NODE_ENV=development
 FRONTEND_URL=http://localhost:5173
+
+RPC_URL=http://127.0.0.1:8545
+CONTRACT_ADDRESS=0x5FbDB2315678afecb367f032d93F642f64180aa3
+PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The PRIVATE_KEY key should go before the RPC_URL key

(UnorderedKey)